assets/dev/js/editor/editor-base.js

```diff
@@ -439,7 +440,11 @@ export default class EditorBase extends Marionette.Application {
 		}
 
 		if ( elementorCommon.config.experimentalFeatures[ 'floating-buttons' ] ) {
-			this.modules.linksPageLibraryModule = new LinksPageLibraryModule();
+			this.modules.floatingButtonsLibraryModule = new FloatingButtonsLibraryModule();
```

Comment:
Just making sure there will be no Core/Pro dependency issues with this renaming?

Reply:
@louiswol94 no I don't see any dependency issue, this is just to update the naming.